### PR TITLE
Change shop construction to no longer auto-rotate to queue paths

### LINKED ISSUE #7424: Shops can auto-rotate towards queue lines
**OS:** Windows 7
**Version:** 0.1.3
**Commit/Build:** a1e0e63

Shops can auto-rotate towards a queue line instead of a regular path. Since shops facing queue lines are useless the auto-rotate function should prioritize normal paths.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

### PR DESCRIPTION
**Pull request by tobyhinloopen**
Fixes #7424

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Fix: [#7382] Opening the mini-map reverts the size of the land tool to 1x1, regardless of what was selected before.
 - Fix: [#7402] Edges of neigbouring footpaths stay connected after removing a path that's underneath a ride entrance.
 - Fix: [#7436] Only the first 32 vehicles of a train can be painted.
+- Fix: [#7424] Auto-rotate shops ignore queue lines
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: We only put fixes in the changelog to bugs that were in the previous version. I think auto-rotate shop was merged after the release of 0.1.2.

Comment (tobyhinloopen) [reply distribution/changelog.txt]: I 🗑 the line

File: contributors.md [hunk 2]
```diff
@@ -74,6 +74,7 @@ The following people are not part of the project team, but have been contributin
 * Joshua Moerman (Jaxan) - Misc.
 * Nicolas Hawrysh (xp4xbox) - Misc.
 * Albert Morgese (Fusxfaranto) - Misc.
+* Toby Hinloopen (tobyhinloopen) - Improvements to auto-rotate of shops
```
Comment (Gymnasiast) [inline contributors.md]: This is really just a bugfix.

**Comment by tobyhinloopen** [comment]
🎉

**Pull request merged**